private_gpt/components/llm/custom/sagemaker.py

```diff
@@ -26,6 +27,8 @@
         CompletionResponseGen,
     )
 
+log = logging.getLogger(__name__)
```

Comment:
tiny NIT but could you rename this to `logger` (and in the other file too)? 

Will be more consistent naming wise for the future.

Reply:
Sure sorry - I'm used to Java's Lombok `@Log4j2` that injects `log` variable into classes 😅 

I personally preferring `log` as it's shorter than `logger` (it makes lines shorter and seems more readable to me), but no problem, changing to `logger`